core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -138,6 +142,7 @@
             + KEY_IS_PAGED + " INTEGER DEFAULT 0,"
             + KEY_NEXT_PAGE_LINK + " TEXT,"
             + KEY_HIDE + " TEXT,"
+            + KEY_SORT_ORDER + " INTEGER DEFAULT " + CODE_UNSPECIFIED + ","
```

Comment:
Please use `SortOrder.CODE_UNSPECIFIED` instead.

Also, I am wondering if it might be a good idea to use a String instead. That would allow to have a comma separated list of sort orders later. If we use an integer, we later need to convert the database and delete the column again.